src/core/Akka/Event/LogLevel.cs

```diff
@@ -31,6 +31,11 @@ public enum LogLevel
         /// The error log level.
         /// </summary>
         ErrorLevel,
+        
+        /// <summary>
+        /// The off log level.
+        /// </summary>
+        OffLevel = int.MaxValue
```

Comment:
Added OffLevel to LogLevel enum, this would act as a marker for the off state

Reply:
I'd revert this change - it's a major part of the public API and will require an update to all downstream loggers probably.